	
<launch>
    <arg name="model_name" />
    <arg name="base_frame_id" />
    <arg name="global_frame_id" />
    <arg name="tf_time_delay" />

    <node pkg="gazebo_tools" type="gazebo_tf_publisher.py" name="gazebo_tf_publisher" output="screen">
        <param name="model_name"					    value="$(arg model_name)" />
        <param name="base_frame_id"				        value="$(arg base_frame_id)" />
        <param name="global_frame_id"				    value="$(arg global_frame_id)" />
        <param name="tf_time_delay"				        value="$(arg tf_time_delay)" />
	</node>
</launch>